### ISSUE TITLE
Prism Breaks Jira Server When Browsing Issue Filter Results

### ISSUE THREAD

**Issue by libantema** [catawiki/Prism#19]
When I browse tickets with a Jira issue filter where the first ticket in the list of results contains a {code} block, the content of the page below the search form is grayed out and I can't click on anything. Disabling the Prism app resolves this issue, confirming that Prism is the culprit.

**Jira Version**: Server v8.0.2
**Prism Version**: v1.2.4
**How to reproduce**: Navigate to URL - for example: `<base_jira_url>/browse/<ticket_with_a_code_block>?filter=-1`
**Current workaround**: Navigate to the ticket without the filter - for example: `<base_jira_url>/browse/<ticket_with_a_code_block>`

If I inspect the page, I see the following JavaScript error in the console (line numbers are probably unique to my instance).
```
Uncaught RangeError: Invalid array length
    at batch.js:6731
    at Object.run (batch.js:6731)
    at Object.highlightElement (batch.js:6731)
    at Object.highlightAllUnder (batch.js:6731)
    at Object.highlightAll (batch.js:6731)
    at HTMLDocument.<anonymous> (batch.js:6734)
    at HTMLDocument.dispatch (batch.js:65)
    at HTMLDocument.v.handle (batch.js:65)
    at Object.trigger (batch.js:65)
    at Object.c.event.trigger (batch.js:85)
```
I navigated to the JS file and set some breakpoints to try to figure out what was going on. I also referenced the [prism.js file](https://github.com/catawiki/Prism/blob/master/src/main/resources/js/prism.js).
What I discovered is that `env.vars['command-line']` (see lines 9231-9232) is an empty object `{}`.
This then breaks the array instantiation on line 9250.
`var promptLines = new Array(commandLine.numberOfLines + 1);`
Some logic for checking the commandLine object before using it will probably fix this issue (and potentially others that haven't been discovered yet).

**Comment by libantema** [comment]
I created an [upstream PrismJS pull request](https://github.com/PrismJS/prism/pull/1862) as a potential fix for this issue. However, the fix will need to be tested for the Prism Jira app.

**Comment by siavashs** [comment]
Nice work @libantema 
We have 2 options:
- wait for your PR to be merged in upstream and release a new app version when a new PrismJS version is out
- patch our copy of PrismJS

**Comment by libantema** [comment]
@siavashs my upstream PR has been merged and Prism 1.16.0 has been built/released. You should be good to update the core PrismJS and test the new Jira app.

**Comment by siavashs** [comment]
@libantema I opened #24 for this update, I hope this will fix the issue.
